Samples/HelloWorld.NetCore/NuGet.Config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="Orleans-local-vNext" value="..\..\vNext\Binaries\Debug\" />
+    <remove key="Orleans-local" />
```

Comment:
Yeah... it was after playing with it for a while. I did notice it before pushing and decided to keep it, as ideally I would just want to remove the clear, but it did not work at least with my local setup.
Agreed on adding MyGet feed, it's just that it's outdated, and currently I noticed that the build scripts for nuget packages were modified, so if we build locally, it doesn't append a version that is higher than what was published to myget 2 months ago. I actually wanted to sync up with you on that.